modules/health_quest/app/controllers/health_quest/v0/questionnaire_responses_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module V0
+    class QuestionnaireResponsesController < HealthQuest::V0::BaseController
+      def index
```

Comment:
```suggestion
      def index
        session_service = HealthQuest::SessionService.new(user)
        map_query = HealthQuest::PatientGeneratedData::QuestionnaireResponse::MapQuery.build(session_service.headers)
        render json: map_query.get(author: user.icn)
      end
```

Any reason this uses a class?  I don't see it reused anywhere.

Reply:
Hi @erluti, thank you for your much-valued suggestion. Please see https://github.com/department-of-veterans-affairs/vets-api/pull/5330#discussion_r526341365